libs/blockscout-service-launcher/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "blockscout-service-launcher"
-version = "0.5.1"
+version = "0.5.2"
```

```suggestion
version = "0.6.0"
```

I would say that it is rather major update. Addition of new public field may potentially be breaking in case users accessed those fields directly (see [`stats-server`](https://github.com/blockscout/blockscout-rs/blob/main/stats/stats-server/src/settings.rs#L53))